about/faq.md

```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
 The [RDK WPE page](https://wiki.rdkcentral.com/display/RDK/WPE) provides more
 information about WPE in the RDK platform.
 
-In 2017 engineers from Igalia submitted a new flavor of WPE, suitable for upstream
-hosting under the webkit.org umbrella. This version of WPE its [released every 6
-months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org).
-This flavor of WPE is maintained upstream and provides regular [security updates](/security).
+In 2017, engineers from [Igalia](https://igalia.com) submitted a new flavor of WPE, suitable for upstream hosting under the [webkit.org](https://webkit.org/) umbrella. This version of WPE is [released every 6 months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org). This flavor of WPE is maintained upstream and provides regular [security updates](/security).
```

Comment:
A question: is the `[upstream repository]` link best pointed to `https://webkit.org` or is there a better destination to use?

Reply:
And the project plans to migrate to GitHub, that might be a better, future-proof link as well.

Reply:
https://github.com/WebKit/WebKit

Reply:
Link updated and pushed.